tests/test_cases/issue_330/issue_330.py

```diff
@@ -45,4 +54,13 @@ async def issue_330_iteration(dut):
         tlog.info("Found %s" % member._path)
         count += 1
 
-    assert count == 2, "There should have been two members of the structure"
+    # Riviera-PRO 2022.10 and newer does not discover inout_if correctly over VPI (gh-3587)
+    rv_2022_10_plus = RivieraVersion(cocotb.SIM_VERSION) >= RivieraVersion("2022.10")
+    if (
+        SIM_NAME.startswith("riviera")
+        and rv_2022_10_plus
```

Comment:
Weird that this is the only one that isn't inlined, but it also doesn't matter.

Reply:
Yeah, but it's a different exception than the other waiver. I could have widened the waiver to all Exception objects, but found that too course grained. That's why we have this solution.

Reply:
Oh sorry, you meant the variable! The autoformatting looks really strange if I inline that, that's why I pulled it out and chose the rather strange variable name.